src/views/profile/PasswordAndSSHKey/SSHKey/SSHKeyCreateUpdate.vue

```diff
@@ -75,7 +75,13 @@ export default {
       }
     }
   },
-  mounted() {
+  computed: {
+    isCreateAction() {
+      return this.meta?.action === 'create'
+    }
+  },
+  async mounted() {
+    this.meta = await this.$store.dispatch('common/getDrawerActionMeta')
     setTimeout(() => {
       this.loading = false
     })
```

Comment:
There do not appear to be any apparent irregularities, potential issues, or optimization suggestions in the provided JavaScript code snippet. The functionality and logic within the given code appears to work correctly and effectively. However, it's recommended that you thoroughly test your application across various scenarios to ensure it behaves as expected under different conditions.